vars/deploy-on-openstack.yml

```diff
@@ -1,15 +1,16 @@
 ---
 deploy_on_openstack_cloudname: "{{ lookup('env','OS_CLOUD') | default('engcloud-cloud-ci', true) }}"
-deploy_on_openstack_prefix: "{{ lookup('env','PREFIX') }}" #PREFIX is set at all times (ensured in script library)
-deploy_on_openstack_external_network: "floating"
-deploy_on_openstack_internal_network: "{{ deploy_on_openstack_prefix }}-net"
-deploy_on_openstack_internal_subnet: "{{ deploy_on_openstack_prefix }}-subnet"
+# PREFIX must be unique per environment.
+# If you want to deploy two environments on the same network,
+# make sure PREFIX is defined for your environment before running run.sh
+deploy_on_openstack_prefix: "{{ lookup('env','PREFIX') | default('socok8s', true) }}"
+deploy_on_openstack_external_network: "{{ lookup('env', 'EXTERNAL_NETWORK') | default('floating', true) }}"
+deploy_on_openstack_internal_network: "{{ lookup('env','INTERNAL_NETWORK') | default('ccpci-net', true) }}"
+deploy_on_openstack_internal_subnet: "{{ lookup('env', 'INTERNAL_SUBNET') | default('ccpci-subnet', true) }}"
```

Comment:
I wonder if we shouldn't give defaults to these at the risk of developers deploying over each other. Perhaps we should raise an error if the env is unset?

Reply:
I kept the current behaviour, but I totally think we should remove those defaults in a later commit.